operator/v1alpha1/types_image_content_source_policy.go

```diff
@@ -24,14 +24,19 @@ type ImageContentSourcePolicySpec struct {
 	// repositoryDigestMirrors allows images referenced by image digests in pods to be
 	// pulled from alternative mirrored repository locations. The image pull specification
 	// provided to the pod will be compared to the source locations described in RepositoryDigestMirrors
-	// and the image may be pulled down from any of the repositories in the list instead of the
+	// and the image may be pulled down from any of the mirrors in the list instead of the
 	// specified repository allowing administrators to choose a potentially faster mirror.
 	// Only image pull specifications that have an image disgest will have this behavior applied
 	// to them - tags will continue to be pulled from the specified repository in the pull spec.
-	// When multiple policies are defined, any overlaps found will be merged together when the mirror
-	// rules are written to `/etc/containers/registries.conf`. For example, if policy A has sources `a, b, c`
-	// and policy B has sources `c, d, e`. Then the mirror rule written to `registries.conf` will be `a, b, c, d, e`
-	// where the duplicate `c` is removed.
+	//
+	// Each “source” repository is treated independently; configurations for different “source”
```

Comment:
nit: I think it should be mentioned that `source` across ICSP's will be as one unit independently.

Reply:
Ahh, i see it's mentioned here https://github.com/openshift/api/pull/384/files#diff-6ab961f49a89df71eeb35043bc6e638cR35